spec/support_specs/basic_auth_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe 'Require basic authentication', type: :request do
+  include TestHelpers::BasicAuthHelper
+
+  before do
+    stub_const('FEATURES', FEATURES.merge(basic_auth: { enabled: true, username: 'basic', password: 'auth' }))
```

Comment:
If we're using this mechanism for feature flags (I quite like it) could we possibly wrap it in a friendlier interface?

Reply:
We have `climate_control` for manipulating ENV in specs

Reply:
If you like the direction (of having some simple feature flags defined in this way), I'll add an interface. How about something like:

```
Features.enabled?(:basic_auth)
Features.get(:basic_auth, :username)
```

Reply:
Implemented the suggested interface, but called it BasicAuth instead of Features